tests/DS/sds_detect_version/test_detect_version.sh

```diff
@@ -20,7 +20,15 @@ function test_oscap_info {
 	stderr="$(mktemp)"
 	ds="$(mktemp)"
 	cp $srcdir/scap-ds.xml $ds
-	sed -i "s/X.X/${version}/g" $ds
+
+	case $(uname) in
```

Comment:
How about adding `xsed` function to the `test_common.sh` and using `sed` or `gsed` there based on `uname`?